src/components/Calendar/index.js

```diff
@@ -135,6 +137,12 @@ export default class Calendar extends Component {
     }
 }
 
+export default function Calendar(props) {
+    return <Consumer>{values => <CalendarComponent {...values} {...props} />}</Consumer>;
```

Comment:
pass only the locale from context since other value will come here and Calendar only need the locale